.gitignore

```diff
@@ -2,7 +2,6 @@
 !.*
 
 composer.phar
-composer.lock
```

Comment:
Please undo this

Reply:
In past we decided to track 'composer.lock' file, right?

Reply:
Right, I was confused. Yes, we decided to track it in https://phabricator.wikimedia.org/T359146 where we forgot to list this repo and I mistook this line for deciding to *not* track it, sorry. We keep this change and thanks for asking!